import.html

```diff
@@ -20,6 +20,7 @@
     <script src="./js/libs/vendors/js-beautify/beautify-html.min.js"></script>
     <script src="./js/libs/vendors/babel-polyfill/polyfill.js"></script>
     <script src="./js/libs/vendors/exceljs/exceljs.min.js"></script>
+    <script src="./js/libs/vendors/xpath-to-css/xpath-to-css.js"></script>
```

Comment:
Won't this be done in the blueprint detector code?  I'm getting confused about how this is progressing.

Reply:
Depending on how mapping object creation progresses this can hopefully be removed. It is currently the last ditch effort when there is not selector, domId or domClasses. IIUC, however there should always be a `selector` property provided by the blueprint detector which would enable all of this logic to be removed.